Sources/BraintreeCoreSwift/BTClientTokenError.swift

```diff
@@ -0,0 +1,53 @@
+import Foundation
+
+///  Error codes associated with a client token.
+@objc public enum BTClientTokenError: Int, Error, CustomNSError, LocalizedError {
```

Comment:
I actually don't think we want these error classes to be public. I think we want our merchant to just access the returned `NSError` or `Error` type, instead of our custom types. (We do that in [this PR](https://github.com/braintree/braintree_ios/pull/762/files)).

Reply:
Currently the BTHTTP tests are in Obj-C. If we need to unit test internal Swift methods in Obj-C tests, we will have to write our tests in Swift. We can certainly do that in this PR since we know the Obj-C tests pass, but that is why this is currently public.

We currently have ~7 Obj-C test files in Core and most (if not all) are for the remaining conversions. We can certainly convert the tests along with the files, but will likely end up with some ~1k+ line conversions for the remaining PRs.